README.md

```diff
@@ -4,28 +4,20 @@ Mattermost is an open source, self-hosted Slack-alternative https://mattermost.o
 
 It's written in Golang and React and runs as a single Linux binary with MySQL or Postgres. Every month on the 16th [a new compiled version is released under an MIT license](https://about.mattermost.com/download/).
 
-This project hosts plugins built by Mattermost staff and community. They are categorized as follows:
+[index.md](index.md) is the list of plugins built by Mattermost staff and community. They are categorized as follows:
 
  - **Production**: Certified by the Mattermost team, and tested prior to each Mattermost server release.
  - **Unofficial**: Non-production plugins built by Mattermost staff and community.
 
 ### Add your plugin
 
-To add your plugin as an unofficial plugin:
+To add your plugin as an unofficial plugin submit a pull request with a reference of your plugin included in [index.md](index.md)
```

Comment:
Why do we keep the list in a different file then `README.md`? I think this just adds more complexity and people have to find and click on a different file, then the one they landed on first. I propose to just have everything in `README.md`. 3/5

Reply:
I almost agree with it. My only reason for placing it into another file was to simplify bot-parsing it, just scan for URL-like links. README.md contains other links so we'd need to include another parseable indication of what are parseable plugins. (An easy counterpoint - any URL that points to a github repo and has a `plugin.json` is a plugin, so who cares what other links there are, just a tad more crawling)

Reply:
Let's leave it as a separate file for now. Jason's right, if we get the appstore soon enough this shouldn't matter